cnd/src/http_api/route_factory.rs

```diff
@@ -155,15 +155,8 @@ pub fn create(facade: Facade, allowed_origins: &AllowedOrigins) -> BoxedFilter<(
         .and(warp::path!("dial"))
         .and(warp::path::end())
         .and(warp::body::json())
-        .and(facade.clone())
-        .and_then(orderbook::post_dial_peer);
-
-    let post_announce_trading_pair = warp::post()
```

Comment:
Removing this will break the e2e test, please adapt accordingly.

Reply:
Not the e2e test on dev :) Thanks for the heads up though I'll keep in mind when working on the e2e test.